site-structure.json

```diff
@@ -114,16 +113,56 @@
+                    { "pageName": "Creating a Primary Schema", "href": "/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingAPrimarySchema.html" },
+                    { "pageName": "Creating Panels", "href": "/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingPanels.html" },
+                    { "pageName": "Creating Enactors", "href": "/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.html" },
+                    { "pageName": "Creating an Auxiliary Schema", "href": "/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingAnAuxiliarySchema.html" },
+                    { "pageName": "Instantiating the Preferences Editor", "href": "/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/InstantiatingThePreferencesEditor.html" },
+                    { "pageName": "Instantiating the Enhancer and Settings Store Only", "href": "/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/InstantiatingTheEnhancerAndSettingsStoreOnly.html" },
+                    { "pageName": "Styling the Preferences Editor", "href": "/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/StylingThePreferencesEditor.html" },
                     { "pageName": "Adding a Basic Preferences Editor to a Site", "href": "/to-do/AddingPrefsEditorToSite.html" },
-                    { "pageName": "Working With A Preferences Editor On Your Site", "href": "/to-do/WorkingWithPrefsEditorOnSite.html" },
+                    { "pageName": "Working With A Preferences Editor On Your Site", "href": "/to-do/WorkingWithPrefsEditorOnSite.html" }
+                ]
+            },
+            {
+                "sectionName": "Creating an Adjuster UI",
+                "pages":
+                [
                     { "pageName": "Creating a New Adjuster UI", "href": "/tutorial-creatingANewAdjusterUI/CreatingANewAdjusterUI.html" },
-                    { "pageName": "How to Create and Use Font Icons", "href": "/tutorial-creatingANewAdjusterUI/HowToCreateAndUseFontIcons.html" },
-                    { "pageName": "Creating a preferences Editor Using the Preferences Framework", "href": "/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingAPreferencesEditorUsingThePreferencesFramework.html" },
+                    { "pageName": "How to Create and Use Font Icons", "href": "/tutorial-creatingANewAdjusterUI/HowToCreateAndUseFontIcons.html" }
+                ]
+            },
+            {
+                "sectionName": "User Interface Options",
+                "pages":
+                [
+                    { "pageName": "User Interface Options", "href": "/tutorial-userInterfaceOptions/UserInterfaceOptions.html" }
+                ]
+            },
+            {
+                "sectionName": "",
```

Comment:
The string templates tutorial does not look like it's in its own section, it looks like it's in the UIO section. I think we should move it to the Prefs Framework section; what do you think?